SimPEG/Examples/EM_FDEM_SusEffects.py

```diff
@@ -0,0 +1,148 @@
+from SimPEG import *
+from SimPEG import EM
+from pymatsolver import MumpsSolver
+from scipy.constants import mu_0
+
+def run(plotIt=True):
+    """
+        EM: FDEM: Effects of susceptibility
+        ===================================
+
+        When airborne freqeuncy domain EM (AFEM) survey is flown over
+        the earth including significantly susceptible bodies (magnetite-rich rocks),
+        negative data is often observed in the real part of the lowest frequency
+        (e.g. Dighem system 900 Hz). This phenomenon mostly based upon magnetization
+        occurs due to a susceptible body when the magnetic field is applied.
+
+        To clarify what is happening in the earth when we are exciting the earth with
+        a loop source in the frequency domain we run three forward modelling:
+
+            - F[:math:`\sigma`, :math:`\mu`]: Anomalous conductivity and susceptibility
+            - F[:math:`\sigma`, :math:`\mu_0`]: Anomalous conductivity
+            - F[:math:`\sigma_{air}`, :math:`\mu_0`]: primary field
+
+        We plot vector magnetic fields in the earth. For secondary fields we provide
+        F[:math:`\sigma`, :math:`\mu`]-F[:math:`\sigma`, :math:`\mu_0`]. Following
+        figure show both real and parts.
+
```

Comment:
@sgkang : could you add the link to EM GeoSci and or em_examples where this lives? Thanks!